pkgs/applications/misc/ArchiSteamFarm/default.nix

```diff
@@ -23,7 +23,10 @@ buildDotnetModule rec {
 
   dotnet-runtime = dotnetCorePackages.aspnetcore_6_0;
 
-  nugetDeps = if stdenvNoCC.isAarch64 then ./deps-aarch64-linux.nix else ./deps-x86_64-linux.nix;
+  nugetDeps = ./deps.nix;
+
+  # Without this dotnet attempts to restore for Windows targets, which it cannot find the dependencies for
+  dotnetRestoreFlags = [ "--runtime linux-${if stdenvNoCC.hostPlatform.isAarch64 then "arm64" else "x64"}" ];
```

Comment:
We shouldn't just fallback to x64. We should also check that and error if the platform is anything else.

Reply:
We should probably have something similar to `toRustTarget` in our Rust infra to translate a platform name to the .Net Runtime ID.

Reply:
That also sounds like a good idea to reduce code repetition.

Reply:
> We should probably have something reusable similar to toRustTarget in our Rust infra to translate a platform name to the .Net Runtime ID.

I wonder what the right location for that would be. Maybe a passthru attribute from the SDK? We could also create a `dotnetPlatform` attribute, could include stuff like `mkNugetSource` as well.

Reply:
Added it to the SDK for now.